CONFIGDOC.md

```diff
@@ -8,15 +8,15 @@ You can use the Jenkins&trade; plugin for MATLAB&reg; in freestyle and multi-con
 -  [Set Up Multi-Configuration Project](#set-up-multi-configuration-project)
 
 ## Use MATLAB for Build
-Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox appears in the **Build Environment** section of the project configuration window.
+Once you install the plugin on Jenkins, **Use MATLAB version** appears in the **Build Environment** section of the project configuration window.

```

Comment:
Should it be "Once you install the plugin **in** Jenkins, ..."? 

"on Jenkins" sounds weird to me.

Reply:
Changed it to "on your Jenkins agent".